src/NuGet.Clients/VsExtension/source.extension.vs15.vsixmanifest

```diff
@@ -30,5 +30,10 @@
     <Asset Type="Microsoft.VisualStudio.Assembly" d:Source="Project" d:ProjectName="NuGet.VisualStudio.Interop" Path="|NuGet.VisualStudio.Interop|" AssemblyName="|NuGet.VisualStudio.Interop;AssemblyName|" />
     <Asset Type="Microsoft.VisualStudio.MefComponent" d:Source="Project" d:ProjectName="NuGet.VisualStudio.Implementation" Path="|NuGet.VisualStudio.Implementation|" />
     <Asset Type="Microsoft.VisualStudio.Assembly" d:Source="Project" d:ProjectName="NuGet.VisualStudio" Path="|NuGet.VisualStudio|" AssemblyName="|NuGet.VisualStudio;AssemblyName|" />
+    <Asset Type="Microsoft.VisualStudio.MefComponent" d:Source="Project" d:ProjectName="NuGet.SolutionRestoreManager" Path="|NuGet.SolutionRestoreManager|" />
+    <Asset Type="Microsoft.VisualStudio.VsPackage" d:Source="Project" d:ProjectName="NuGet.SolutionRestoreManager" Path="|NuGet.SolutionRestoreManager;PkgdefProjectOutputGroup|" />
   </Assets>
+  <Prerequisites>
```

Comment:
Also this seems like a new requirement for vsixmanifests. I could not find any docs on this on the interwebs.